cli/src/template.rs

```diff
@@ -27,6 +28,25 @@ token = "{}"
     )
 }
 
+pub fn idl_ts(idl: &Idl) -> Result<String> {
+    let mut idl = idl.clone();
+    idl.accounts = idl
+        .accounts
+        .into_iter()
+        .map(|acc| {
+            let mut acc = acc;
+            acc.name = acc.name.to_mixed_case();
```

Comment:
Noticed a problem with this only now when started updating the project on which I work. Why do we change the account name to mixed case? @ChewingGlass